### PR TITLE
Test ceph osd low ops alert test 4.12 a

### PR DESCRIPTION
**Pull request by DanielOsypenko**
Cherry pick:
https://github.com/red-hat-storage/ocs-ci/pull/7897

### CODE REVIEW

**Comment by DanielOsypenko** [comment]
> PR validation on existing cluster
> 
> Cluster Name: dosypenk-289 Cluster Configuration: vSphere7-DC-CP_VC1 PR Test Suite: tier3 PR Test Path: tests/manage/monitoring/test_workload_with_distruptions.py::TestCephOSDSlowOps::test_ceph_osd_slow_ops_alert Additional Test Params: OCP VERSION: 4.12 OCS VERSION: 4.12 tested against branch: release-4.12
> 
> [Job FAILED](https://url.corp.redhat.com/4997967) (installation failed, tests not executed).

wrong job parameters

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -3070,3 +3070,22 @@ def check_ceph_cmd_execute_successfully():
 
         logger.warning(f"Failed to execute the ceph command due to the error {str(ex)}")
         return False
+
+
+def get_mon_pod_by_pvc_name(pvc_name: str):
```
Comment (fbalak) [inline ocs_ci/ocs/resources/pod.py:3075]: Is this function needed? It seems that it is not referenced anywhere.

Comment (DanielOsypenko) [reply ocs_ci/ocs/resources/pod.py:3075]: you are right. The reference to it was removed when I changed the test in one of the commits. 
there only 4.10 and 4.11 left to merge, rest where merged with this func. Considering it is not making harm, should I remove it?

Comment (fbalak) [reply ocs_ci/ocs/resources/pod.py:3075]: I think that it would be better to remove it if it is nowhere used.

**Review by fbalak — DISMISSED** [review]

File: tests/manage/monitoring/test_workload_with_distruptions.py [hunk 2]
```diff
@@ -172,3 +182,151 @@ def test_workload_with_checksum_verify(
...
+            pod_obj.run_io(
+                storage_type="fs",
+                size="3G",
+                runtime=self.timeout_sec,
+                fio_filename=f"{pod_obj.name}_io",
+            )
+
+        self.start_workload_time = time.perf_counter()
+
+        def finalizer():
```
[the remark this answers is not in the document]
Comment (DanielOsypenko) [reply tests/manage/monitoring/test_workload_with_distruptions.py:243]: if I will move it to the top of setUp and we fail before `self.pod_objs` list is created & filled up (line: 225, line: 231) the finalizer will throw AttributeError on tearDown call

[the remark this answers is not in the document]
Comment (petr-balogh) [reply tests/manage/monitoring/test_workload_with_distruptions.py:243]: Isn't then the easy fix to move the self.pod_objs = [] to line 193 and right after to define finalizer and add finalizer?

[the remark this answers is not in the document]
Comment (DanielOsypenko) [reply tests/manage/monitoring/test_workload_with_distruptions.py:243]: done, created a fixture set_osd_op_complaint_time in tests/conftest.py

**Comment by DanielOsypenko** [comment]
required update, still relevant

**Pull request closed without merging**